assets/activation-scripts/activate

```diff
@@ -163,6 +175,72 @@ to_eval="$($_flox_activations \
 eval "$to_eval"
 export _FLOX_ACTIVATION_STATE_DIR _FLOX_ACTIVATION_ID
 
+# Now that we support attaching to an environment we can no longer rely on
+# the environment variable replay for setting the PATH and MANPATH variables,
+# and must instead infer them from the FLOX_ENV_DIRS variable maintained for
+# us by the flox CLI.
+
+# Set IFS=: for this portion of the script.
+_save_IFS="$IFS"
+IFS=":"
+
+# Get an iterable array of FLOX_ENV_DIRS.
+declare -a _FLOX_ENV_DIRS
+# shellcheck disable=SC2206
+_FLOX_ENV_DIRS=($FLOX_ENV $FLOX_ENV_DIRS)
```

Comment:
```suggestion
# If there's an outer activation with the CLI followed by an inner activation with just the activate script (this could happen e.g. for a build), we need to combine $FLOX_ENV and $FLOX_ENV_DIRS
# If $FLOX_ENV is already in $FLOX_ENV_DIRS, the deduplication logic below will handle that
_FLOX_ENV_DIRS=($FLOX_ENV $FLOX_ENV_DIRS)
```